recipes/parcels/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "parcels" %}
+{% set version = "1.0" %}
+{% set sha256 = "d9753572a3f099caf5c17b477753034fe8f714788e588874ade18fd91a573f13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/oceanparcels/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+      - remove_setuptools-scm_dependency.patch
+
+build:
+  skip: True  # [py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - python
+    - cachetools >=1.0.0
+    - cgen
+    - enum34
+    - ffmpeg >=3.2.3,<3.2.6
+    - gcc  # [unix]
+    - m2w64-toolchain  # [win]
+    - matplotlib 2.0.2
+    - netcdf4 >=1.1.9
+    - numpy >=1.9.1
+    - progressbar2
+    - python-dateutil
+    - scipy >=0.16.0
+    - six >=1.10.0
+    - xarray >=0.5.1
+test:
+  imports:
+    - parcels
+  requires:
+    - pytest
+
+about:
+  home: http://github.com/OceanParcels/parcels
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: 'Probably A Really Computationally Efficient Lagrangian Simulator'
+
+  description: |
+    Parcels (Probably A Really Computationally Efficient Lagrangian Simulator)
+    is an experimental prototype framework aimed at exploring novel approaches
+    for Lagrangian tracking of virtual ocean particles in the petascale age.
+  doc_url: http://oceanparcels.org/
+  dev_url: https://github.com/OceanParcels/parcels
+
+extra:
+  recipe-maintainers:
+    - erikvansebille
+    - delandmeterp
```

Comment:
@delandmeterp are you ok with being a maintainer?

Reply:
Yes sure!